pkg/aspect/build/build.go

```diff
@@ -29,7 +30,8 @@ const (
 	skipPromptKey = "build.skip_prompt"
 
 	SpecifiedFolderOption = "All targets in a specified folder (path relative to your WORKSPACE)"
-	CurrentFolderOption   = "All targets in current folder"
+	CurrentFolderOption   = "All targets within current folder"
+	CurrentPackageOption  = "All targets within current package (nearest parent if current folder is not a package)"
```

Comment:
I don't think you need the parenthetical - the "current package" is always defined this way